api-docs.js

```diff
@@ -897,6 +897,210 @@ const apiDoc = {
         },
       },
     },
+    '/wallet/utxo-filter': {
+      get: {
+        summary: 'Return utxos and some helpful information regarding it.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            'in': 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          },
+          {
+            name: 'max_utxos',
+            'in': 'query',
+            description: 'Maximum number of utxos to aggregate. Default to MAX_INPUTS (255)',
+            required: false,
+            schema: {
+              type: 'integer',
+            },
+          },
+          {
+            name: 'token',
+            'in': 'query',
+            description: 'Token to filter the utxos. If not sent, we select only HTR utxos.',
+            required: false,
+            schema: {
+              type: 'string',
+            },
+          },
+          {
+            name: 'filter_address',
+            'in': 'query',
+            description: 'Address to filter the utxos.',
+            required: false,
+            schema: {
+              type: 'string',
+            },
+          },
+          {
+            name: 'amount_smaller_than',
+            'in': 'query',
+            description: 'Maximum limit of utxo amount to filter the utxos list. We will consolidate only utxos that have an amount lower than this value. Integer representation of decimals, i.e. 100 = 1.00.',
+            required: false,
+            schema: {
+              type: 'integer',
+            },
+          },
+          {
+            name: 'amount_bigger_than',
+            'in': 'query',
+            description: 'Minimum limit of utxo amount to filter the utxos list. We will consolidate only utxos that have an amount bigger than this value. Integer representation of decimals, i.e. 100 = 1.00.',
```

Comment:
'We will consolidate ...' -> 'We will return ...'

Reply:
Done, thanks!